devel/jsonnet/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        google jsonnet 0.17.0 v
+revision            0
+categories          devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         The data templating language
+
+long_description    A data templating language for app and tool developers
+
+homepage            http://jsonnet.org/
+
+checksums           rmd160  e43a5cee00aaea3782407c9ce0b87d28f34fc3b9 \
+                    sha256  9f4450ff784952a0f5c593ce7dabd83cbda2eed62a4b01cc2ba507b32483c8dd \
+                    size    21859800
+
+use_configure       no
+
+post-extract {
+    # fix destroot commands like `cp $(BINS) $(PREFIX)/bin/`
+    reinplace -E \
+        {s/^([[:space:]]+(cp|mkdir)[[:space:]]+.+[[:space:]]+)\$\(PREFIX\)/\1$(DESTDIR)$(PREFIX)/g} \
+        ${worksrcpath}/Makefile
+}
```

Comment:
Warning: reinplace s/^([[:space:]]+(cp|mkdir)[[:space:]]+.+[[:space:]]+)\$\(PREFIX\)/\1$(DESTDIR)$(PREFIX)/g didn't change anything

Reply:
please use a patch file to make required changes instead of cryptic reinplace statements

Reply:
Thanks for the working patch file below. Done.